families/burrow_evm/src/sawtooth_burrow_evm/cli/show.go

```diff
@@ -95,35 +60,50 @@ func (s *Show) Run() (err error) {
 
 func DisplayEntry(entry *EvmEntry) {
 	if entry == nil {
-		fmt.Println("Entry is nil")
+		fmt.Println("Nothing at that address")
 		return
 	}
 
-	act := entry.GetAccount()
-	stg := entry.GetStorage()
+	acct := entry.GetAccount()
 
-	if len(act.GetAddress()) == 0 {
+	if len(acct.GetAddress()) == 0 {
 		fmt.Println("Account does not exist")
 		return
 	}
 
+	addr := hex.EncodeToString(acct.GetAddress())
+	code := hex.EncodeToString(acct.GetCode())
+
 	fmt.Printf(`
 Address: %v
```

Comment:
This address is being printed out with trailing 0's (which makes it a 35 byte addr). This gives us a malformed address if it is copied into another seth command.